qa/rpc-tests/wallet_shieldcoinbase_ua_nu5.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
The list in `rpc-tests.py` refers to a `wallet_shieldcoinbase_unified.py` that doesn't appear to exist.